packages/ketcher-react/src/style/mixins.less

```diff
@@ -469,3 +469,55 @@
 	font-size: 0.00001px;
 	-webkit-text-size-adjust: none;
 }
+
+.dialog-tabs() {
+	width: 100%;
+	a {
+		padding: 8px 12px;
+		margin-right: 2px;
+		line-height: 14px;
+		display: inline-block;
+		cursor: pointer;
+		text-align: center;
+		background-color: #eee;
+		border: 1px solid transparent;
+		border-radius: 4px 4px 0 0;
+		&:hover {
+			background-color: #9dc2d0;
+		}
+		&.active {
+			border: 1px solid #ddd;
+			border-bottom-color: transparent;
+			background-color: white;
+		}
+	}
+}
+
+.input() {
+	border-radius: 4px;
+	display: inline-block;
+	vertical-align: middle;
+	background: white;
+	line-height: 20 / 14;
+	height: 2.4em;
+	padding: 0.35em 0.15em 0.35em 0.3em;
+	margin: 2px;
+	border: 1px solid @border-color;
+	box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+	box-sizing: border-box;
+	&:active, &:focus {
+		outline: 0;
+	}
+	&:hover, &:focus {
+		.highlight-shadow(black);
+	}
+
+	&[disabled],
+	&[readonly],
+	fieldset[disabled] & {
+		cursor: not-allowed;
+		//box-shadow: none;
+		background: #efefef;
+		opacity: 0.6;
+	}
+}
```

Comment:
incorrect indents, has no line ending

Reply:
Good point, @TomasLight, but I'd prefer to leave indents as is because the whole `mixins.less` file has incorrect indents. This PR is already pretty big, formatting changes in 500+ lines-file will only complicate PR.
Ending line added.

Reply:
But if not now, so when?)
Up to you